com.unity.netcode.gameobjects/Tests/Editor/SnapshotRttTests.cs

```diff
@@ -9,7 +9,7 @@ public class SnapshotRttTests
         [Test]
         public void TestBasicRtt()
         {
-            var snapshot = new SnapshotSystem();
+            var snapshot = new SnapshotSystem(NetworkManager.Singleton);
```

Comment:
So this is an editor test. And I don't see us creating a NetworkManager in this test or in SetUp/Teardown. So where does this NetworkManger.Singleton come from. Pretty sure it is null. But then why is this test not failing?

Reply:
nice catch. As it turns out `snapshot` is only used to `snapshot.GetConnectionRtt`. It doesn't run or anything. The rest is really unit testing the `ConnectionRtt`. As such, it doesn't care to have a `null` NetworkManager.

I think it would be clearer if we did:

`var snapshot = new SnapshotSystem(default);`

And then, it would still compile when the Singleton gets removed in the future.